nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/AvgPooling3D.java

```diff
@@ -51,6 +50,11 @@ public AvgPooling3D(SameDiff sameDiff,INDArray arrayInput, INDArray arrayOutput,
         super(sameDiff, null, new INDArray[]{arrayInput}, wrapOrNull(arrayOutput), false, config, Pooling3DType.AVG);
     }
 
+    public AvgPooling3D(INDArray[] input, Pooling3DConfig pooling3DConfig) {
+        super(null,null,input,null,false,pooling3DConfig,Pooling3DType.AVG);
```

Comment:
As per the dl4j-dev-tools PR: this should be `AvgPooling3D(@NonNull INDArray input, ...` not `INDArray[]`